tests/scene/test_code_edit.h

```diff
@@ -3445,6 +3445,98 @@ TEST_CASE("[SceneTree][CodeEdit] completion") {
 		}
 	}
 
+	SUBCASE("[CodeEdit] autocomplete suggestion order") {
+		/* favorise less fragmented suggestion */
+		code_edit->clear();
+		code_edit->insert_text_at_caret("te");
+		code_edit->set_caret_column(2);
+		code_edit->add_code_completion_option(CodeEdit::CodeCompletionKind::KIND_VARIABLE, "test", "test");
+		code_edit->add_code_completion_option(CodeEdit::CodeCompletionKind::KIND_VARIABLE, "tset", "tset");
+		code_edit->update_code_completion_options();
+		code_edit->confirm_code_completion();
+		CHECK(code_edit->get_line(0) == "test");
+
+		/* favorise suggestion starting from the string to complete (matching start)*/
```

Comment:
Make sure to keep a space before the closing `*/` (we typically wouldn't use `/* */` comment style for single line comments like this but there's a precedent in this file so it's fine to keep it consistent).